projects/igniteui-angular/src/lib/grids/filtering/grid-filtering.service.ts

```diff
@@ -38,7 +38,7 @@ export class IgxFilteringService implements OnDestroy {
     private columnToExpressionsMap = new Map<string, ExpressionUI[]>();
     private filterPipe = new IgxGridFilterConditionPipe();
     private titlecasePipe = new TitleCasePipe();
-    private datePipe = new DatePipe(window.navigator.language);
+    private datePipe = new IgxDatePipeComponent(window.navigator.language);
```

Comment:
I think the date pipe should use `grid.locale` instead of `window.navigator.language`.